src/graphql/generated.ts

```diff
@@ -373,7 +373,7 @@ export enum ColonyActionType {
   CreateDomain = 'CREATE_DOMAIN',
   /** An action related to creating a domain within a Colony via a motion */
   CreateDomainMotion = 'CREATE_DOMAIN_MOTION',
-  /** An action related to creating expenditure (dvanced payment)  */
+  /** An action related to creating expenditure (dvanced payment) */
```

Comment:
Tiny typo

Reply:
Good spot, think I fixed it in the schema so just need to re-run codegen 👍